sda-commons-dependency-check/src/test/java/org/sdase/commons/dependency/check/UnwantedDependenciesTest.java

```diff
@@ -126,6 +126,21 @@ void checkForApacheHttpClientV4() {
     }
   }
 
+  /**
+   * This test finds the library `com.github.tomakehurst` in our classpath. We don't want to use it,
```

Comment:
nit: the library is missing `wiremock` at the end. You can see it below in your implementation.